cmd/connector/config/config.toml

```diff
@@ -36,10 +36,8 @@
     # Defines the number of active persisters to keep open
     NumPersistersToKeep = 2
 
-    # Defines the first commitable meta block round
-    # If it is set to 0, it will try to get last rounds info from storage,
-    # otherwise, it will try to sync from observers
-    FirstCommitableBlock = 30
+    # Defines the first commitable meta block nonce
+    FirstCommitableBlock = 0
```

Comment:
maybe also for the other shards have explicitly the first commitable block

Reply:
added first commitable blocks per shard